src/constants.ts

```diff
@@ -86,3 +86,59 @@ export const MARKET_DEPTH_END_PRICE_SLIPPAGE_PERC = 20;
 
 // Logging
 export const NUMBER_SOURCES_PER_LOG_LINE = 12;
+
+// ABI for balance checker
+export const BALANCE_CHECKER_ABI = [
```

Comment:
Given that this contract now exists in `Protocol` and has 0x bindings, should we use the 0x tooling instead of `ethers`?

Reply:
👍